docs/FAQ.md

```diff
@@ -0,0 +1,29 @@
+# Frequently Asked Questions (FAQ)
```

Comment:
This is a great idea. Can we add this FAQ as a link to the README for increased visibility?

Reply:
SGTM. Will follow-up w/ a separate PR soas not to unnecessarily delay merging this incremental value.